recipes/geographiclib/all/conanfile.py

```diff
@@ -132,12 +127,17 @@ def package(self):
         ]:
             rmdir(self, os.path.join(os.path.join(self.package_folder, folder)))
         rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+        if not self.options.tools:
+            rmdir(self, os.path.join(self.package_folder, "sbin"))
+            bin_files = [it for it in os.listdir(os.path.join(self.package_folder, "bin")) if not it.endswith(".dll")]
+            for it in bin_files:
+                rm(self, it, os.path.join(self.package_folder, "bin"))
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "geographiclib")
         self.cpp_info.set_property("cmake_target_name", "GeographicLib::GeographicLib")
         self.cpp_info.set_property("pkg_config_name", "geographiclib")
-        self.cpp_info.libs = collect_libs(self)
+        self.cpp_info.libs = ["GeographicLib"] if Version(self.version) >= "2" else ["Geographic"]
```

Comment:
@uilianries : With conan 1.64.0 getting error 
`-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19045.
-- Conan: Target declared 'GeographicLib::GeographicLib'
CMake Error at build/generators/cmakedeps_macros.cmake:39 (message):
  Library 'GeographicLib' not found in package.  If 'GeographicLib' is a
  system library, declare it with 'cpp_info.system_libs' property
Call Stack (most recent call first):
  build/generators/geographiclib-Target-debug.cmake:24 (conan_package_library_targets)
  build/generators/geographiclibTargets.cmake:26 (include)
  build/generators/geographiclib-config.cmake:16 (include)
  CMakeLists.txt:4 (find_package)`

But if I revert it then working fine. Haven't tested it with Conan 2.x.